target/linux/ramips/image/mt76x8.mk

```diff
@@ -240,6 +240,15 @@ define Device/jotale_js76x8-32m
 endef
 TARGET_DEVICES += jotale_js76x8-32m
 
+define Device/kroks_kndrt31r19
+  IMAGE_SIZE := 14m
```

Comment:
Please use 15040k instead.

Reply:
I'm not sure if we really need such a big image as I removed `memory` partition in the end (1MB, extended `firmware` to the same amount), so the tail now is `rootfs_data`:
```
root@OpenWrt:~# cat /proc/mtd 
dev:    size   erasesize  name
mtd0: 00030000 00010000 "u-boot"
mtd1: 00010000 00010000 "u-boot-env"
mtd2: 00010000 00010000 "factory"
mtd3: 00fb0000 00010000 "firmware"
mtd4: 001eb18d 00010000 "kernel"
mtd5: 00dc4e73 00010000 "rootfs"
mtd6: 00a30000 00010000 "rootfs_data"

Reply:
> Please use 15040k instead.

I've removed `memory` partition (it only served vendor's needs and unused normally) and thus extended `firmware` to 1MiB.
As seen above, the new `firmware` size is 00fb0000 which is 16064k, this value is used as IMAGE_SIZE.
Please reopen if my calculations are wrong.